### ISSUE TITLE
Update Sociable facet icon

### ISSUE THREAD

**Issue by RueShejn** [Yellow-Dog-Man/Resonite-Issues#3776]
> Might as well chip in my two cents here,
> 
> 
> 
> I used an [online color-vision deficiency](https://www.color-blindness.com/coblis-color-blindness-simulator/) simulator to contrast how the Online and Sociable status look to somebody with color blindness. I figured since both Hero Green and Hero Blue have a similar hue, saturation, and luminosity, they could be difficult to distinguish for those with degraded blue perception. Sure enough, the two colors are rather difficult, if not nearly impossible to distinguish without accompanying labels. Below are the simulated results:
> 
> 
> 
> ### Normal vision
> 
> ![StatusColorContrastNormal](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/7483e76c-2214-48e8-bc2c-70cec520eb12)
> 
> 
> 
> ### Blue-Weak/Tritanomaly: Tough to distinguish
> 
> ![StatusColorContrastTritanomaly](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/23686140-cea5-4981-9770-6aa401ab1783)
> 
> 
> 
> ### Blue-Blind/Tritanopia: Nearly identical
> 
> ![StatusColorContrastTritanopia](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/7e47ff54-15f7-4203-b7e8-b2462cb5753f)
> 
> 
> 
> ### Monochromacy/Achromatopsia: Indistinguishable
> 
> ![StatusColorContrastAchromatopsia](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/34515928-7cc0-4a21-a70b-7585219c7fb1)
> 
> 
> 
> So, I agree that an icon is needed to distinguish the two statuses, like Away, Busy, and Invisible. I mocked up some prototype icons, and like several others in this thread, I think that a 4-point star is probably the best option in terms of distinguishability, symbolism, and simplicity. 
> 
> 
> 
> ### Stars
> 
> ![SociableStars](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/c2dd6a63-d983-4aa3-bbdf-670071fdda2f)
> 
> As I said before, these are the simplest. Easy to recognize, apart from the 5-point star, which might be interpreted as a "favorite" or otherwise special contact by a new user. These should also be readable at a small scale.
> 
> 
> 
> ### Dots
> 
> ![SociableDots](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/c022be1c-b6fb-4bc3-b143-3254de57bb85)
> 
> I guess these could be thought of as representing a conversation circle, but I don't think they offer enough visibility, especially at smaller scales.
> 
> 
> 
> ### Symbols
> 
> ![SociableSymbols](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8518150/a6c4709b-27a6-4c6b-962f-504144a934ea)
> 
> These intentionally convey the point, but when compared to the other status icons they are too noisy. They could also be mistaken for meaning you have a new message, or you're being invited. (Comment icon from Nucleo Core icons, Waving Hand modified from Twemoji)
> 
> 
> 
> On a minor side note, as brought up by lxw404 earlier in the thread, having your current status be Online makes it the same shape as Invisible. I've not tested the contrast through colorblindess filters, but the lower luminosity of Hero Purple should make it different enough from Hero Green. Still, another point of accessability worth considering. 

 _Originally posted by @djsime1 in [#2173](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2173#issuecomment-2151296576)_

**Comment by shininghero** [comment]
Definitely a good idea, but it should be done for all the status icons.  Here's a screenshot of what MS Teams has, as a reference.

![Image](https://github.com/user-attachments/assets/cff9f23f-0b32-4a92-8ac0-81d80e239d5e)

As for a sociable status icon, I'll need a paint program after work to try my idea, but I'm picturing a central circle and 3 or 5 smaller ones around it, with lines to the small ones.

**Comment by shiftyscales** [comment]
The other statuses already all have their own shapes, @shininghero.

Online is a full circle, away is a half circle, busy is a cross, and invisible is a ring / empty circle.

![Image](https://github.com/user-attachments/assets/c43a0c73-1e9c-4c0a-aa4c-acd9dd60a438)

**Comment by ChromaShiba** [comment]
![Image](https://github.com/user-attachments/assets/8d24d04f-b1ba-4840-9412-813e3218f21d) This icon now exists, not too much we can do right now in terms of colors but the shape is different. This will reflect in the latest press kit off the website and internal files

![Image](https://github.com/user-attachments/assets/bb4dfc43-d29a-4980-82b2-cfb6661854c0)

**Comment by RyuviTheViali** [comment]
I've added the new icon to the online status facet which should appear for new users getting the facets for the first time and folks who reset their dash. A facet preset version bump still needs to be done for this change to apply to all existing online status preset facets.

![Image](https://github.com/user-attachments/assets/641295fb-2025-49c9-ba51-cdb2a984b5b4)

**Comment by Redd56** [comment]
id say complete right?

**Comment by AegisTheWolf** [comment]
Closing this Issue as the new facet with the new Icon has been updated to the default dash.
